src/EthernaIndex.Swarm/DtoModel/MetadataVideoDto.cs

```diff
@@ -29,13 +27,12 @@ public class MetadataVideoDto
 #pragma warning restore CS8618 // Non-nullable field must contain a non-null value when exiting constructor. Consider declaring as nullable.
 
         // Properties.
-        public string Id { get; set; }
-        public string? Title { get; set; }
         public string? Description { get; set; }
-        public string? OriginalQuality { get; set; }
-        public string? OwnerAddress { get; set; }
-        public float? Duration { get; set; }
+        public float Duration { get; set; }
+        public string OriginalQuality { get; set; }
+        public string OwnerAddress { get; set; }
+        public IEnumerable<MetadataVideoSourceDto> Sources { get; set; }
         public SwarmImageRawDto? Thumbnail { get; set; }
-        public IEnumerable<MetadataVideoSourceDto>? Sources { get; set; }
+        public string Title { get; set; }
```

Comment:
when use JsonSerializer.Deserialize<MetadataVideoDto> some variable could be NULL (in this case of invalid manifest)